sites/topomath.asu.edu/modules/system_descriptions/system_descriptions.module

```diff
@@ -70,6 +72,7 @@ function createSpecificBlocks(){
 							</div>';
 	return $specific_blocks;
 }
+
```

Comment:
I mean is hard coded html required, as far as I see it it consists of a text and two buttons in a modal. I am pretty sure it would be possible to use something from Drupal.